resolwe_bio/descriptors/super_sample.yml

```diff
@@ -0,0 +1,263 @@
+- slug: super_sample
+  name: Super sample annotation
+  description: Sample meta data compliant with GEO and ENCODE databases.
+  version: 0.0.1
+  schema:
+    - name: assay_type
+      label: Assay type
+      type: basic:string
+      choices:
```

Comment:
If this is to be the new default descriptor schema for sample object, this list is not comprehensive enough. Please see this [example](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/descriptors/reads.yml#L12) of the available choices in the GEO system. Explore also the latest and most up-to-date annotation $ submission template that can be dowloaded from GEO site. Use the `allow_custom_choice: true` for the choices menus where this is applicable.

Reply:
We added the option `Other` instead of custom user input.